ls1intum/Artemis
# `Assessment`: Disable override assessment button when the assessment was not changed

### Checklist
#### General



#### Client

### Motivation and Context


Closes https://github.com/ls1intum/Artemis/issues/4592

### Description
Override Assessment button is disabled until changes are made.

### Steps for Testing

Prerequisites:
- 1 Instructor

1. Log in to Artemis
2. Navigate to Submissions page of a File Upload, Modeling and Programming Exercise
3. Open Assessment
4. Verify that the Override Assessment button is initially disabled and is enabled when a change is made in the assessment.

### Review Progress





#### Code Review
#### Manual Tests

### Test Coverage






### Screenshots


![image](https://user-images.githubusercontent.com/67344158/150694653-8f312ef2-2df1-4bf0-a50c-56d3b7d97f88.png)
![image](https://user-images.githubusercontent.com/67344158/150694665-56ff08fd-f6ba-453a-8910-64a4026258e4.png)
